src/app/components/client/ExposureCard.tsx

```diff
@@ -155,7 +159,10 @@ const ScanResultCard = (props: ScanResultCardProps) => {
             </dt>
             <dd>
               <span
-                className={`${styles.exposureCompanyTitle} ${styles.companyNameArea}`}
+                className={`${styles.exposureCompanyTitle} ${
+                  styles.companyNameArea
+                }
+                ${isCompanyNameTooLong && styles.makeFontSmaller}`}
```

Comment:
If `isCompanyNameTooLong` is falsy this will return `null` so a ternary condition that returns just an empty string would be preferable.